taxi_service/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for taxi_service project.
+Generated by 'django-admin startproject' using Django 5.1.3.
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-%r=h#=rck&hwbb(9m$1k1%kgn5yqs!cc93%k4wn51#(-q--au1'
```

Comment:
The `SECRET_KEY` is hardcoded and should be kept secret in production. Consider using environment variables or a separate configuration file to manage sensitive information.